packages/idyll-document/package.json

```diff
@@ -15,7 +15,7 @@
     "build": "npm run build:cjs && npm run build:es",
     "dev": "yarn run build:es --watch",
     "prepublishOnly": "npm run build",
-    "test": "jest"
+    "test": "jest --updateSnapshot"
```

Comment:
We don't want to pass `--updateSnapshot` every time, it will prevent tests from ever failing. That should either be a manual step or an separate script.

Reply:
Oh yeah thanks - I forgot to remove that!